meta/main.yml

```
@@ -1,16 +1,22 @@
+---
 galaxy_info:
   author: Lars Kiesow
   role_name: opencast_postgresql
   namespace: elan
   company: ELAN e.V.
   description: Install and prepare PostgreSQL for Opencast
   license: BSD-3-Clause
-  min_ansible_version: 2.9
+  min_ansible_version: "2.9"
+
   galaxy_tags:
     - opencast
+
   platforms:
     - name: EL
-      versions:
-        - 8
+      versions: ["8", "9"]
```

Set list values in array notation like this is fine, but I would prefer to have it in a yaml notation (as list items). It is easier to read, diff and keep style consistent, IMO. Can you update this please?

```suggestion
      versions:
        - "8"
        - "9"
```